front/src/modules/study/components/StudyCard.tsx

```diff
@@ -29,8 +29,8 @@ export default function StudyCard({ setFilterChips, study }: Props) {
   };
 
   return (
-    <div className="studies-list-card">
-      <div className="studies-list-card-name" data-testid={study.name}>
+    <div className="studies-card">
```

Comment:
Rename studies-card to study-card in all the classes
```suggestion
    <div className="study-card">
```